app/events/listener.go

```diff
@@ -146,10 +146,6 @@ func (l *TelegramListener) Do(ctx context.Context) error {
 			if update.Message == nil {
 				continue
 			}
-			if update.Message.Chat == nil {
-				log.Print("[DEBUG] ignoring message not from chat")
-				continue
-			}
 
```

Comment:
the removed check for update.Message.Chat make me worry

Reply:
This case doesn't exist anymore as Chat is not a pointer and I have no idea how to trigger it.